forms.py

```diff
@@ -28,12 +42,12 @@ class PewSheetForm(FlaskForm):
     feasts = Feast.upcoming()
     feast_choices = [(feast.slug, feast.name) for feast in feasts]
     primary_feast = SelectField(
-        'Primary Feast',
+        'Primary Feast',  #choices=[]
         choices=feast_choices,
     )
     secondary_feasts = SelectMultipleField(
         'Secondary Feasts',
-        choices=[('', '')] + feast_choices,
+        choices=[('', '')] + feast_choices, validate_choice=False
```

Comment:
Why no validation?

Reply:
Otherwise, if the newly added feast is selected from the dropdown, 'Not a valid choice' is displaying. The workaround to this is no validation, see https://github.com/pallets-eco/wtforms/issues/434.